esbuild.config.mjs

```diff
@@ -15,7 +15,6 @@ const globals = {
   'rss-parser': 'Parser',
   'react-mailchimp-subscribe': 'ReactMailchimpSubscribe',
   'plotly.js-basic-dist': 'Plotly',
-  '@rjsf/core': 'JSONSchemaForm',
```

Comment:
Let's include RJSF in the bundle. Bundle size doesn't change much since we were bundling our forked version, and now we're dropping it.